plugins/UserId/lang/en.json

```diff
@@ -4,6 +4,7 @@
         "UserReportTitle": "User IDs",
         "UserReportDocumentation": "This report shows visits and other general metrics for every individual User ID.",
         "PluginDescription": "Shows user reports",
-        "VisitorsUserSubcategoryHelp": "The User IDs report shows visits associated with all of your registered and logged in users. You can use this section to understand website usage by specific users and identify who your most and least active users are."
+        "VisitorsUserSubcategoryHelp": "The User IDs report shows visits associated with all of your registered and logged in users. You can use this section to understand website usage by specific users and identify who your most and least active users are.",
+        "ThereIsNoDataForThisReport": "There is no data for this report.<br/><br/>To generate this report %1$s Read more about this topic in the online guide. %2$s "
```

Comment:
be good if we reused the existing translation for `There is no data for this report.`. This way it will be translated already for most users. Like you can do something like this below (not sure what the correct translation keys are)

```
$view->config->no_data_message .= '<br><br>' . sprintf(Piwik::translate('UserId_ThereIsNoDataForThisReport');
// or 
$view->config->no_data_message = Piwik::translate('General_NoDataReport...') . '<br><br>' . sprintf(Piwik::translate('UserId_ThereIsNoDataForThisReport');

```